components/dot_menu/dot_menu.tsx

```diff
@@ -371,6 +373,12 @@ export class DotMenuClass extends React.PureComponent<Props, State> {
             this.props.handleDropdownOpened(false);
             break;
 
+        // save / unsave
+        case Utils.isKeyPressed(e, Constants.KeyCodes.S):
+            this.handleFlagMenuItemActivated(e);
+            this.props.handleDropdownOpened(false);
+            break;
```

Comment:
@jfrerich I'm seeing an inconsistently working shortcut keys in post menu, especially if you triggered particular key more than once. Like in a case where you saved a post but then change to unsaved.
Eliminating code below at `onShortcutKeyDown` seemed to solve the issue but I wonder why it's placed in the first place.
```javascript
if (this.keysHeldDown.includes(e.key)) {
    return;
}
this.keysHeldDown.push(e.key);
```